common/config/presets/collectibles.toml

```diff
@@ -0,0 +1,6 @@
+[collectibles]
```

Comment:
This could go beneath the banner to be in line with the other configs?

Reply:
Done, I did it based on validator node config, which is this way. Will change that as well.